app/views/content/life-as-a-teacher/age-groups-and-specialisms/secondary/_article.html.erb

```diff
@@ -0,0 +1,58 @@
+<div class="row inset">
+  <section class="col col-720 col-space-l-top">
+    <h2 class="heading--box-blue">Why teach <%= @front_matter["subject"] %></h2>
+    <div class="inset">
+      <p>Becoming a secondary teacher is an excellent career choice for people who want to make an impact. Your expertise and love for your subject could spark the interests of your pupils to follow in your footsteps. Beyond the classroom, you'll also play a crucial role in building pupils' confidence and supporting their personal growth.</p>
+      <!-- Why teach: benefits -->
+      <%= render 'content/shared/teaching/benefits' %>
+    </div>
+
+    <h2 class="heading--box-blue"> Who you'll teach</h2>
+    <h3>Age groups you could teach</h3>
+    <p>Children start secondary school when they’re 11. There are up to 3 stages in secondary school:</p>
+    <ul>
+      <li>key stage 3 (KS3) includes years 7 to 9, with children aged 11 to 14</li>
+      <li>key stage 4 (KS4) includes years 10 and 11, with pupils aged 14 to 16</li>
+      <li>key stage 5 (KS5) includes years 12 and 13, with students aged 16 to 18 at secondary schools with sixth forms, or colleges</li>
+    </ul>
+    <%= render 'content/shared/qualifications-training/teaching_send' %>
+    <h2 class="heading--box-blue">What you'll teach</h2>
+     <p>As a secondary teacher, you will train as a subject specialist and follow the agreed curriculum for your chosen subject and age group.</p>
+     <p><a href="/life-as-a-teacher/explore-subjects">Explore some subjects you could teach</a>.</p>
+     <%= render 'content/shared/teaching/ages_abilities' %>
```

Comment:
Love that you've done a SEND partial! I think this might need changing a bit though, I'll let you know when I've updated the SEND page and we can tweask